### ISSUE TITLE
config task problem

### ISSUE THREAD

**Issue by JayChan-USTC** [tianweiy/CenterPoint#232]
Hi,author! In model config, the tasks are as follows:
tasks = [
    dict(num_class=1, class_names=["car"]),
    dict(num_class=2, class_names=["truck", "construction_vehicle"]),
    dict(num_class=2, class_names=["bus", "trailer"]),
    dict(num_class=1, class_names=["barrier"]),
    dict(num_class=2, class_names=["motorcycle", "bicycle"]),
    dict(num_class=2, class_names=["pedestrian", "traffic_cone"]),
]

Why do you want to set it that way, instead of directly dividing it into the following forms:

tasks = [
    dict(num_class=1, class_names=["car"]),
    dict(num_class=1, class_names=["construction_vehicle"]),
    dict(num_class=1, class_names=[ "construction_vehicle"]),
    dict(num_class=1, class_names=["bus"]),
    dict(num_class=1, class_names=["trailer"]),
    dict(num_class=1, class_names=["barrier"]),
    dict(num_class=1, class_names=["motorcycle",]),
    dict(num_class=1, class_names=[ "bicycle"]),
    dict(num_class=1, class_names=["pedestrian"]),
    dict(num_class=1, class_names=["traffic_cone"]),
]

**Comment by tianweiy** [comment]
each dict specifies a head and during the training the loss is only backpropagated to the specific head. Basically, if the loss is related to a "car", it will only train the "car" head. 

Therefore, we want to group related objects together instead of separating them completely. This setup is from https://arxiv.org/abs/1908.09492 and det3d.
